### PR TITLE
rename functions in _decorators.py

### PR DESCRIPTION
**Pull request by adriangb**
I tried to figure out what these functions do based on their name and couldn't. I am suggesting these names as alternatives, only if it's clearer for others as well.

### CODE REVIEW

File: pydantic/_internal/_decorators.py [hunk 1]
```diff
@@ -309,7 +309,9 @@ def inspect_validator(validator: Callable[..., Any], mode: FieldValidatorModes)
     )
 
 
-def inspect_field_serializer(serializer: Callable[..., Any], mode: Literal['plain', 'wrap']) -> tuple[bool, bool]:
+def does_field_serializer_require_info_arg(
```
Comment (samuelcolvin) [inline pydantic/_internal/_decorators.py:312]: This function returns two things.

**Review by samuelcolvin — COMMENTED** [review]
Humm, I'm neutral. I agree "inspect" is generic, but these functions don't just decide one thing, they also raise errors if the signature is wrong.

File: pydantic/_internal/_decorators.py (hunk 1 above)
Comment (adriangb) [reply pydantic/_internal/_decorators.py:312]: Good point. It’ll be nice when we can do `-> {“foo”: int}`.

**Comment by adriangb** [comment]
Probably not worth it

**Pull request closed without merging**